container-registry/task-execute-in-dind-cluster.yaml

```diff
@@ -32,6 +32,9 @@ spec:
       description: (optional) the kubernetes cluster namespace where the docker engine is hosted/deployed
       default: "build"
     #
+    - name: registry-create-namespace
+      description: create container registry namespace if it doesn't already exists
+      default: true
```

Comment:
We put default value as plain string in the YAML as we hit some issues with tekton definitions in the past for boolean value
So I would expect `default: "true"`